R/oc_forward.R

```diff
@@ -31,58 +34,89 @@
+#'   returned. Default is \code{NULL}).
+#' @param no_annotations Logical vector indicating whether additional
 #'   information about the result location should be returned. \code{TRUE} by
 #'   default, which means that the output will not contain annotations.
-#' @param no_dedupe A logical vector (default \code{FALSE}), when \code{TRUE}
+#' @param no_dedupe Logical vector (default \code{FALSE}), when \code{TRUE}
 #'   the output will not be deduplicated.
-#' @param no_record A logical vector (default \code{FALSE}), when \code{TRUE} no
-#'   log entry of the query is created and the geocoding request is not cached
+#' @param no_record Logical vector (default \code{FALSE}), when \code{TRUE} no
+#'   log entry of the query is created, and the geocoding request is not cached
 #'   by OpenCage.
-#' @param abbrv A logical vector (default \code{FALSE}), when \code{TRUE}
+#' @param abbrv Logical vector (default \code{FALSE}), when \code{TRUE}
 #'   addresses in the \code{formatted} field of the results are abbreviated
 #'   (e.g. "Main St." instead of "Main Street").
-#' @param add_request A logical vector (default \code{FALSE}) indicating whether
+#' @param add_request Logical vector (default \code{FALSE}) indicating whether
 #'   the request is returned again with the results. If the \code{return} value
 #'   is a \code{df_list}, the query text is added as a column to the results.
 #'   \code{json_list} results will contain all request parameters, including the
-#'   API key used! For \code{geojson_list} this is currently ignored by
-#'   OpenCage.
+#'   API key used! This is currently ignored by OpenCage if return value is
+#'   \code{geojson_list}.
 #' @param ... Ignored.
 #'
-#' @return \code{oc_forward} returns, depending on the \code{return} parameter,
-#'   a list with either
+#' @return Depending on the \code{return} argument, \code{oc_forward} returns a
+#'   list with either
 #'   \itemize{
 #'   \item the results as tibbles (\code{"df_list"}, the default),
 #'   \item the results as JSON lists (\code{"json_list"}),
 #'   \item the results as GeoJSON lists (\code{"geojson_list"}), or
 #'   \item the URL of the OpenCage API call for debugging purposes
 #'   (\code{"url_only"}).
 #'   }
-#'   \code{oc_forward_df} returns a tibble.
 #'
-#' @seealso \code{\link{oc_reverse}} for reverse geocoding, and the
-#'   \href{https://opencagedata.com/api}{OpenCage API documentation} for more
-#'   information about the API.
+#' @seealso \code{\link{oc_forward_df}} for inputs as a data frame, or
+#'   \code{\link{oc_reverse}} and \code{\link{oc_reverse_df}} for reverse
+#'   geocoding. For more information about the API and the various parameters,
+#'   see the \href{https://opencagedata.com/api}{OpenCage API documentation}.
 #'
 #' @export
 #'
 #' @examples
 #' \dontrun{
-#' oc_forward(placename = "Sarzeau")
-#' oc_forward(placename = "Islington, London")
+#' # Geocode a single location, an address in this case
 #' oc_forward(placename = "Triererstr 15,
```

Comment:
Let's better format this according to OC's instructions
```suggestion
#' oc_forward(placename = "Triererstr 15, 99432, Weimar, Deutschland")
```